clog/handlers.py

```diff
@@ -88,6 +89,38 @@ def emit(self, record):
             self.handleError(record)
 
 
+class MonkHandler(logging.Handler):
+    """Handler for sending python standard logging messages to a monk
+        stream.
+
+        .. code-block:: python
+
+            import clog.handlers, logging
+            log = logging.getLogger(name)
+            log.addHandler(clog.handlers.MonkHandler('client_id', 'localhost', 3600, 'stream'))
+
+
+        :param client_id: client id to identify the user logging
+        :param stream: name of the monk stream logs will be sent to
+        :param host: hostname of monk server
+        :param port: port number of monk server
+        """
+
+    def __init__(self, client_id, stream, host=None, port=None):
+        logging.Handler.__init__(self)
+        self.stream = stream
+        self.logger = MonkLogger(client_id, host, port)
+
+    def emit(self, record):
+        try:
+            msg = self.format(record)
+            self.logger.log_line(self.stream, msg)
```

Comment:
At the moment with the file-based stream configuration we can use prefixes to configure the BE  / autocreated streams. We should add a configurable prefix here, so that every stream sent through monk will have a format like `_clog.<stream_name>`.